translations/round_structure.tex/pl.po

```diff
@@ -80,6 +80,27 @@ msgid ""
 "Keep playing new Rounds until any of the Scenario's ending conditions have been met.\n"
 "\n"
 msgstr ""
+"Rozgrywka podzielona jest na Rundy, podczas których każdy gracz wykonuje swoją Turę w kolejności zgodnej z ruchem wskazówek zegara, zaczynając od gracza rozpoczynającego grę.\n"
+"W swoich Turach gracze przemieszczają swoich \\hyperlink{Heroes}{Bohaterów} na Mapie Gry, budują nowe Budynki w swoich \\hyperlink{Town}{Miastach} i rekrutują \\hyperlink{Units}{Jednostki}, próbując spełnić warunek zwycięstwa Scenariusza.\\par\n"
+"\\textbf{Na początku każdej Rundy}, z wyjątkiem pierwszej, wykonaj następujące kroki :\n"
+"\\begin{itemize}\n"
+" \\item Obróć na aktywną stronę wszystkie użyte Żetony Budowy, Populacji i/lub Księgi Zaklęć.\n"
+" \\item Obróć na aktywną, zieloną stronę użyte Żetony Ruchu (PR).\n"
+" \\item Odzyskaj możliwość korzystania z Efektów Mistrzowskich  \\includesvg[height=10px]{\\svgs/expert.svg}.\n"
+"\\end{itemize}\n"
+"Następnie, w zależności od aktualnego Numeru Rundy, gracze zdobywają zasoby lub rozpatrują Kartę Astrologowie Ogłaszają:\n"
+"\\begin{itemize}\n"
+" \\item Rundy o numerach nieparzystych to \\textbf{Rundy Zasobów}.\n"
+" Wszyscy gracze otrzymują przychód z \\hyperlink{Mines}{Budynków Miasta, Osad i Kopalni}, które kontrolują.\n"
```

Comment:
```suggestion
" Wszyscy gracze otrzymują przychód z \\hyperlink{Kopalnie}{Budynków Miasta, Osad i Kopalni}, które kontrolują.\n"
```

Reply:
W nawiasie jest mowa o "budynkach miast, osad i kopalń", co sugeruje, że kategoria "mines" niekoniecznie powinna być dosłownie przetłumaczona na "kopalnie"

Reply:
To tylko technikalia. W drugim nawiasie jest tekst, który będzie hiperłączem. A w pierwszym jest nazwa hiperłącza, czyli jak się nazywa definicja tego pojęcia. W tym momencie może to dla ciebie nie mieć sensu, bo ta definicja pojawia się dopiero w ostatnim rozdziale :D